app/models/user.py

```diff
@@ -8,6 +8,8 @@
 
 class Permission:
     GENERAL = 0x01
+    VENDOR = GENERAL | 0x02  # == 0x03 (will match GENERAL && VENDOR)
```

Comment:
I think this needs to be `VENDOR = 0x02` (and same below). Take a look at page 112 in the Flask book (screenshot for reference)
![screen shot 2015-10-21 at 11 05 04 am](https://cloud.githubusercontent.com/assets/6806102/10649864/6447d416-7814-11e5-8eca-756dcc98bb33.png)